src/Integration/Aws/DynamoDbSession/src/SessionHandler.php

```diff
@@ -66,6 +76,11 @@ public function __construct(DynamoDbClient $client, array $options)
         $options['hash_key'] = $options['hash_key'] ?? 'id';
         $options['session_lifetime_attribute'] = $options['session_lifetime_attribute'] ?? 'expires';
         $options['id_separator'] = $options['id_separator'] ?? '_';
+        $options['consistent_read'] = $options['consistent_read'] ?? true;
+        $options['locking'] = (bool) ($options['locking'] ?? false);
+        $options['max_lock_wait_time'] = (float) ($options['max_lock_wait_time'] ?? 10.0);
+        $options['min_lock_retry_microtime'] = (int) ($options['min_lock_retry_microtime'] ?? 10000);
```

Comment:
I'm not sure such thing deserve parameters.

Reply:
As I understand DynamoDB usually is billed per-request, so giving a possibility to fine-tune these according to your use case may be useful in terms of balancing the functionality & cost-savings. For example, if your sessions usually are locking for several seconds, you likely don't want to be polling every 10-50ms.

Also, these settings are just copied from aws-sdk to make a migration to async-aws as small change as possible.